accountpermissions/css/style.css

```diff
@@ -1,15 +1,15 @@
 body {
-	background: #000;
```

Comment:
hmm that kinda makes this whole PR meaningless :P cuz mostly its this patch. but I guess I will do that